ast/variable_declaration.go

```diff
@@ -151,7 +151,12 @@ func (d *VariableDeclaration) Doc() prettier.Doc {
 		)
 	}
 
-	valueDoc := d.Value.Doc()
+	var valueDoc prettier.Doc
```

Comment:
This is probably temporary. Needed this to make it easy to debug the generated code for now.
See the comment below (in `desugar.go`) as to why the "value" can be `nil` here.

Reply:
Maybe leave a comment, so we do not revert this at some point

Reply:
Added https://github.com/onflow/cadence/pull/3741/commits/9e2d8404367c1cfa12e2e511d2e03ccd263a19c5